resources/views/components/instagram.blade.php

```diff
@@ -0,0 +1,11 @@
+<div class="insta-content">
+    <div class="insta-link"><a href="javascript:void(0)" rel="me" target="_blank" class="">{{ trans('app.follow_me') }}!</a></div>
+    <div id="ri-grid" class="ri-grid ri-grid-size-2">
+        <img class="ri-loading-image" src="{{ asset(config('images_path.instagram_loading_gif')) }}" alt="" />
+        <ul>
+            @for($i = 1; $i < 21; $i++)
```

Comment:
```suggestion
            @for ($i = 1; $i < 21; $i++)
```
đưa số vào config đi anh :v

Reply:
Số này chỉ là ở đây anh muốn hiển thị demo giao diện bao nhiêu bài thôi